### ISSUE TITLE
Add ability to close tab by double clicking it

### ISSUE THREAD

**Issue by RDKRACZ** [brave/brave-browser#31202]
A double click to close tab feature is present in Vivaldi (off by default afaik).

**Comment by rleyvasal** [comment]
I agree this would be a great addition to Brave, specially when using vertical tabs.

**Comment by K0-RR** [comment]
Please, it should be a really simple thing to implement. It was requested more than a year ago.

**Comment by MrTHP** [comment]
still waiting for this feature in 2025
